src/core.js

```diff
@@ -0,0 +1,38 @@
+// TODO: non-working code. Implement after accepting an API.
+
+var modules = {};
+
+function registerModule(command, moduleObject) { 
+    modules[command] = moduleObject;
+}
+
+function unregisterModule(command) {
+    if (modules[command].enabled) {
+        modules[command].onDisable();
+    }
+    delete modules[command];
+}
+
+function handleCommand(command, args) {
+    if (modules[command].onReceive === undefined) {
+        return -1; // throw NoOnReceiveMethodException()
+    }
+    if (!modules[command].enabled) {
```

Comment:
Can we ignore disabled modules here?

Reply:
To avoid an accidental removing of the function from the object, I'm considering to leave checking of the onReceive function availability, even when we won't allow adding a module without it.